stella-boleto/src/test/java/br/com/caelum/stella/boleto/BoletoTest.java

```diff
@@ -39,7 +39,7 @@ public void fatorVencimentoComDataMuitoAntiga() {
 		Datas.novasDatas().comDocumento(01, 01, 1996).comProcessamento(01, 1, 1996).comVencimento(1, 2, 1996);
 	}
 
-	@Test(expected = IllegalArgumentException.class)
+	@Test
```

Comment:
Seria legal criar um teste para o novo cenário, ao invés de só remover a exception. Assim podemos garantir que datas novas vão ser pegas pelo teste. E se for manter esse teste, é importante adicionar um assert.

Reply:
Ok, obrigado pelas dicas.